components/organisms/Register/RegisterFormPassword/RegisterFormPassword.module.css

```diff
@@ -20,11 +19,6 @@
   color: var(--white);
 }
 
-.registerFormPassword-buttonSessionDisabled:hover {
-  background-color: var(--white);
-  color: var(--turquoise);
-}
-
 .registerFormPassword-buttonDisabled {
```

Comment:
Change "registerFormPassword-buttonDisabled" to registerFormPassword-disabledButton

Reply:
fixed